app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -31,6 +31,7 @@ export class EWSAccount extends MailAccount {
   readonly tls = TLSSocketType.TLS;
   readonly canSendInvitations: boolean = false;
   readonly folderMap = new Map<string, EWSFolder>;
+  calendar: EWSCalendar | void;
```

Comment:
Can you please remove this?

I'm trying to keep the zig-zack dependencies at bay. Email should not depend on Calendar.

(The code below that creates the calendar should also eventually move out of here. E.g. into setup. One hack doesn't justify another.)

Reply:
Yes, please. Shared calendars are coming right after.